### PR TITLE
Гладышев - Лабораторная работа #1

### PR DESCRIPTION
**Pull request by KJlaccHoeUM9l**

### CODE REVIEW

**Comment by KJlaccHoeUM9l** [comment]
@Olegka35
@DaniilRoman

File: code/line_and_plane_intersection/model/intersection.py [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+    def have_intersection(line: Line, plane: Plane) -> bool:
+        line_point, line_direction = line.get_describe()
+        plane_abcd = plane.get_describe()
+        plane_normal = plane_abcd[:3]
+        is_parallel = np.dot(line_direction.to_tuple(), plane_normal) == 0
+
+        if not is_parallel:
+            return True
+        else:
+            if plane.point_on_plane(line_point):
```
Comment (DaniilRoman) [inline code/line_and_plane_intersection/model/intersection.py:114]: maybe we can return plane.point_on_plane(line_point) ?

**Comment by DaniilRoman** [comment]
:+1:

**Review by Olegka35 — APPROVED** [review]
👍

**Review by AntonGrishin — APPROVED** [review]

**Pull request merged**